packages/framework/build.xml

```diff
@@ -0,0 +1,1029 @@
+                <exclude name="/"/>
+            </fileset>
+            <fileset dir="${path.web.scripts}/">
+                <exclude name="/"/>
+            </fileset>
+        </delete>
+    </target>
+
+    <target name="clean-redis" depends="redis-check" description="Cleans up redis cache">
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true" output="${dev.null}">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:redis:clean-cache"/>
+        </exec>
+    </target>
+
+    <target name="clean-redis-old" depends="redis-check" description="Cleans up redis cache for previous build versions">
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true" output="${dev.null}">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:redis:clean-cache-old"/>
+        </exec>
+    </target>
+
+    <target name="redis-check" description="Checks availability of Redis">
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:redis:check-availability"/>
+        </exec>
+    </target>
+
+    <target name="clean-styles" description="Cleans up directories with CSS generated by Grunt.">
+        <delete failonerror="false" includeemptydirs="true">
+            <fileset dir="${path.web.styles.admin}/">
+                <exclude name="/"/>
+            </fileset>
+            <fileset dir="${path.web.styles.front}/">
+                <exclude name="/"/>
+            </fileset>
+        </delete>
+    </target>
+
+    <target name="composer" description="Installs dependencies for production.">
+        <exec executable="${path.composer.executable}" logoutput="true" passthru="true" checkreturn="true">
+            <arg value="install"/>
+            <arg value="--no-dev"/>
+        </exec>
+    </target>
+
+    <target name="create-domains-data" depends="create-domains-db-functions" description="Creates domains data for newly configured domains.">
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:domains-data:create"/>
+        </exec>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:recalculations"/>
+        </exec>
+    </target>
+
+    <target name="create-domains-db-functions" description="Creates new domains DB functions." hidden="true">
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:domains-db-functions:create"/>
+        </exec>
+    </target>
+
+    <target name="product-search-create-structure" description="Creates structure for searching via elasticsearch.">
```

Comment:
I was thinking this could be useful if you wanted to create a structure eg. with a different prefix (and then delete the old prefix). But as I'm thinking about it I would use the Symfony commands `shopsys:product-search:delete-structure` and `shopsys:product-search:export-products` as they give me more control around the prefixes using ENV variables...